### ISSUE TITLE
Guidance alert box removal

### ISSUE THREAD

**Issue by jpetersononrr** [ONRR/onrr.gov-site#658]
Hi!

Can you remove the executive order alerts at the top of these pages?  Thanks!

https://onrr.gov/reportpay/policy.htm
https://onrr.gov/reportpay/handbooks/index.htm
https://onrr.gov/DearRep.htm
https://onrr.gov/reportpay/training/index.htm
https://onrr.gov/unbundling/index.htm
https://onrr.gov/valuation/index.htm

**Comment by Maroyafaied** [comment]
@jpetersononrr can you specify exactly what needs to be removed? Should I take out that whole white box? Thanks 
![image](https://user-images.githubusercontent.com/40175688/114928710-24cc1b00-9df0-11eb-97de-18e441447bf7.png)

**Comment by jpetersononrr** [comment]
Yes please!!  Each of these:

***@***.***D73692.482CCE20]

​Jodie Peterson
Minerals Revenue Specialist
Office of Natural Resources Revenue
***@***.***
(303)231-3719

Royalty Valuation - Consistent Answers to Complex Questions

From: Maroyafaied ***@***.***>
Sent: Thursday, April 15, 2021 1:41 PM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Peterson, Jodie F ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Guidance alert box removal (#658)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@jpetersononrr<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjpetersononrr&data=04%7C01%7Cjodie.peterson%40onrr.gov%7C36278cb724f749c8ab9908d90046754e%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637541124884869161%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=eraAEOkpsJkB1%2BZ%2FHCmMJ%2BSQ9hNh4O8ue1cfKIXwmzo%3D&reserved=0> can you specify exactly what needs to be removed? Should I take out that whole white box? Thanks
[Image removed by sender. image]<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F40175688%2F114928710-24cc1b00-9df0-11eb-97de-18e441447bf7.png&data=04%7C01%7Cjodie.peterson%40onrr.gov%7C36278cb724f749c8ab9908d90046754e%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637541124884869161%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=BiTi0eUUT53omMFQbdqcZWWBjmP8yn3U10iNUOMdGXU%3D&reserved=0>

**Comment by Maroyafaied** [comment]
@jpetersononrr please review 

http://2017onrr/reportpay/policy.htm
http://2017onrr/reportpay/handbooks/index.htm
http://2017onrr/DearRep.htm
http://2017onrr/reportpay/training/index.htm
http://2017onrr/unbundling/index.htm
http://2017onrr/valuation/index.htm

**Comment by jpetersononrr** [comment]
That looks great!  Thanks!

​Jodie Peterson
Minerals Revenue Specialist
Office of Natural Resources Revenue
***@***.***
(303)231-3719

Royalty Valuation - Consistent Answers to Complex Questions

From: Maroyafaied ***@***.***>
Sent: Wednesday, April 21, 2021 10:28 AM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Peterson, Jodie F ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Guidance alert box removal (#658)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@jpetersononrr<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjpetersononrr&data=04%7C01%7Cjodie.peterson%40onrr.gov%7C8586e2a06efe4204d82308d904e27eb0%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637546193106323993%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=xIO0GqEUJzHTRs5gXTei%2BmT8zUdQfu042Ng7ct%2FcbIc%3D&reserved=0> please review

http://2017onrr/reportpay/policy.htm
http://2017onrr/reportpay/handbooks/index.htm
http://2017onrr/DearRep.htm
http://2017onrr/reportpay/training/index.htm
http://2017onrr/unbundling/index.htm
http://2017onrr/valuation/index.htm

**Comment by mcharg** [comment]
@Maroyafaied Don't push the training page live until #667 is approved.  I'll push it then.

**Comment by Maroyafaied** [comment]
@mcharg @jpetersononrr all the link should be live in 30 min except this one since we're making other changes to that page for another issue but will be live as well once dev is approved. http://2017onrr/reportpay/training/index.htm

**Comment by Maroyafaied** [comment]
All pages are live. Thanks
